src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -211,8 +210,13 @@ public static function singularify($plural)
             }
         }
 
+        $irregularNouns = array(
```

Comment:
I had it like that prior to the PR but since those are irregular nouns in that context I have moved that into the function.